### ISSUE TITLE
feat(patch): Youtube: Add setting to force original language audio.

### ISSUE THREAD

**Issue by neversleeping** [revanced/revanced-patches#1875]
### Application

YouTube

### Issue

There is a rather new feature in YouTube: different sound languages. When you know multiple languages, it's usually better to watch a video in original one. But it's every time YouTube finds a translation to my native language, it will set it to default.

### Patch

I want a setting that forces original sound in a video, or, at least, selects a language to prioritise.

### Motivation

The most amount of videos have native English sound, and there is a huge amount of people, who's native language is not an English, but they know it good enough to watch a content in original variant.

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by LuigiTrevisan** [comment]
Or maybe something like "Force audio language."
Nonetheless, we need a setting to change the audio language, as the videos are getting automatically translated.

**Comment by Julienraptor01** [comment]
I agree that a settings to automatically pick either the original audio for the vid or to maybe pick the 1st available from a preference list would be very nice

And what i mean by original is the one that is labelled like that by YouTube
Example 1:
![Screenshot_20231112_132502.png](https://github.com/ReVanced/revanced-patches/assets/49841713/3bc8e7b9-e0ae-46fb-844b-e79565f0e3f8)
Example 2:
![Screenshot_20231112_132523.png](https://github.com/ReVanced/revanced-patches/assets/49841713/6ac7eac8-0a64-4f61-a4fb-e419d45838cf)
